extensions/amp-subscriptions/0.1/amp-subscriptions.js

```diff
@@ -21,13 +21,17 @@ import {LocalSubscriptionPlatform} from './local-subscription-platform';
 import {PageConfig, PageConfigResolver} from '../../../third_party/subscriptions-project/config';
 import {Renderer} from './renderer';
 import {SubscriptionPlatform} from './subscription-platform';
-import {dev, user} from '../../../src/log';
 import {installStylesForDoc} from '../../../src/style-installer';
 import {tryParseJson} from '../../../src/json';
+import {user} from '../../../src/log';
 
 /** @const */
 const TAG = 'amp-subscriptions';
 
+
+/** @typedef {{services:!Array<!JsonObject>}} */
+export let ServiceConfig;
```

Comment:
Same discussion, either convert or drop typedef. IMHO, it's just better to store a list of platform configs as a private var.